libsrc/apple2/beep.s

```diff
@@ -0,0 +1,20 @@
+;
+; Colin Leroy-Mira, 2024
+;
+; void beep(void)
+;
+
+        .export         _beep
+        .import         BELL
+
+        .include        "apple2.inc"
+
+        .segment        "LOWCODE"
+
+_beep:
+        lda     CH              ; Bell scrambles CH in 80col mode on IIgs, storing
```

Comment:
I didn't know that. Does this mean that on the IIgs, BELL is more or less a wrapper around COUT with A set to 7 ?